recipe/meta.yaml

```diff
@@ -7,34 +7,51 @@ package:
 source:
   url: https://pypi.io/packages/source/s/scikit-image/scikit-image-{{ version }}.tar.gz
   sha256: dd7fbd32da74d4e9967dc15845f731f16e7966cee61f5dc0e12e2abb1305068c
-  # url: https://github.com/scikit-image/scikit-image/archive/v{{ version }}.tar.gz
-  # fn: scikit-image-{{ version }}.tar.gz
-  # sha256: 3b8414ccb9b524d0d507909e9a4d3843af6ca9919152199d380f2b21639abf17
+  patches:
+    - skip-broken-tests.patch
 
 build:
-  number: 0
+  number: 1
   skip: True  # [py==27]
```

Comment:
Skip py<36, see https://github.com/scikit-image/scikit-image/blob/13a6ec15c0ea8c84abb7ab54b13b97fae67ae44a/setup.py#L233

Reply:
hmm, no.  we will need it for py 37 too, and 0.16.2 was supporting it in the past too.  btw testing passes fine on 3.7 for it